cdc/processor.go

```diff
@@ -523,11 +523,8 @@ func (p *processor) flushTaskStatusAndPosition(ctx context.Context) error {
 			if p.statusModRevision == modRevision && !taskStatus.SomeOperationsUnapplied() {
 				return false, nil
 			}
+			// task will be stopped in capture task handler, do nothing
```

Comment:
Please leave some TODO, add unit tests in the future.

Reply:
This file has no unit tests now, the whole file should be tested in the future. so I think a TODO is not needed here.